vnp/README.md

```diff
@@ -14,9 +14,9 @@ Use the MESH Client script to send the same file multiple times to a given MESH
 
 Example:
 
-    ./mesh.sh send gp_mailbox "@../src/intTest/resources/edifact/pathology.edifact.dat" 3 
+    for ((i = 1 ; i <= 3; i++)); do ./mesh.sh send lab_results_mailbox "@../src/intTest/resources/edifact/pathology.edifact.dat"; done
```

Comment:
This file was renamed to `pathology_3.edifact.dat` (with a `_2` counterpart)

Reply:
I'll update with main branch and check

Reply:
Yeah, `pathology.dat` doesn't exist anymore as we replace it with `pathology_2.edifact.dat` and `pathology_3.edifact.dat` for NHS002 and NHS003 messages.